src/graph.h

```diff
@@ -204,6 +215,8 @@ struct Edge {
   BindingEnv* env_;
   VisitMark mark_;
   size_t id_;
+  int64_t run_time_ms_;
```

Comment:
Why not use accessor methods like `run_time_ms()` and `set_run_time_ms()` for this member? build.cc sets the value directly, which seems wrong. Can you add a comment e.g. "Run time in milliseconds for the edge's command, taken from the build log when present, estimated otherwise."